astrosat_users/models/models_customers.py

```diff
@@ -205,7 +205,8 @@ def invite(self, **kwargs):
         ):
             context["username"] = user_username(user)
 
-        adapter.send_mail(template_prefix, user.email, context)
+        managers_emails = customer.customer_users.managers().values_list("user__email", flat=True)
+        adapter.send_mail(template_prefix, user.email, context, cc=managers_emails)
```

Comment:
What would happen if the `send_mail` failed? Say SMTP server is down.

Reply:
I believe an error would be raised.  If this is done in the context of a view, then a 500 would be returned.  I haven't tested this though.